commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -629,6 +638,13 @@ public int getCommerceOrdersCount(long groupId, long commerceAccountId) {
 		return commerceOrderPersistence.countByG_C(groupId, commerceAccountId);
 	}
 
+	public int countCommerceOrdersByCompanyId(
```

Comment:
Heyyy..... this should actually be a `get...Count` like you had it the first time. Except the `ByCompanyId` is unnecessary -- as long as the signature is unique as `getCommerceOrdersCount(long, long)` then that's the right name. It's only `count*` in the persistence layer.

Also this should return a long.

Also method should've been sorted after the rename (though with the new rename this may be the right spot).

/cc @marco-leo

Reply:
@alecsloan sorry, I asked that change

Reply:
Reverted back. We need ByCompanyId since we have `long groupId, long commerceAccountId`. I did however forget an Override. adding and regenning.

@ethanbustad how come it should return a long? The other count methods return an int and so does the persistence.

Reply:
Oh, never mind then -- my bad :)